examples/Forwarding.go

```diff
@@ -13,32 +15,50 @@ var l3Rules *packet.L3Rules
 
 // Main function for constructing packet processing graph.
 func main() {
+	var err error
 	// Initialize YANFF library at 16 cores by default
 	config := flow.Config{
 		CPUList: "0-15",
 	}
-	flow.SystemInit(&config)
+	if err := flow.SystemInit(&config); err != nil {
+		log.Fatal(err)
+	}
 
 	// Get splitting rules from access control file.
-	l3Rules = packet.GetL3ACLFromORIG("Forwarding.conf")
+	l3Rules, err = packet.GetL3ACLFromORIG("Forwarding.conf")
+	if err != nil {
+		log.Fatal(err)
+	}
 
 	// Receive packets from zero port. Receive queue will be added automatically.
-	inputFlow := flow.SetReceiver(0)
+	inputFlow, err := flow.SetReceiver(uint8(0))
+	if err != nil {
+		log.Fatal(err)
+	}
 
 	// Split packet flow based on ACL.
 	flowsNumber := 5
-	outputFlows := flow.SetSplitter(inputFlow, l3Splitter, uint(flowsNumber), nil)
+	outputFlows, err := flow.SetSplitter(inputFlow, l3Splitter, uint(flowsNumber), nil)
```

Comment:
UES

Reply:
done in the form of
outputFlows, err := flow.SetSplitter(inputFlow, l3Splitter, uint(flowsNumber), nil)
CheckFatal(err)